content/plugins/ralphjsmit-media-library-manager.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Media Library Manager
-slug: ralphjsmit-media-library-manager
+slug: media-library-pro
```

Comment:
The slug needs to match the file name, and contain your author name

Reply:
I updated the slugs back to what they were before the new V3 site update. Otherwise all the URLs on external sites (like [this](https://laravel-news.com/onboarding-manager-pro), [this](https://twitter.com/ralphjsmit/status/1651968224181846020), [this](https://ralphjsmit.com/packages) etc.) to the plugins will return a 404..

Reply:
Let's add redirects in web.php then, I'm aiming for consistency here and trying to reduce clashes when packages have the same name

Reply:
Like this?